src/FileIndexer.ts

```diff
@@ -707,3 +681,46 @@ function isEqualArray<T>(a: T[], b: T[]): boolean {
   }
   return true
 }
+
+function declarationName(node: ts.Node): ts.Node | undefined {
```

Comment:
Extracted into an independent function to satisfy the `class-methods-use-this` ESLint rule.

Reply:
It's a big improvement in readability to have a toplevel function instead of an instance method.